src/sentry/static/sentry/app/views/eventsV2/eventDetails/linkedEvents.tsx

```diff
@@ -72,14 +72,14 @@ class RelatedItems extends AsyncComponent<Props, State> {
     return endpoints;
   }
 
-  renderRelatedIssue() {
+  renderLinkedIssue() {
```

Comment:
If the linked issue is in the sidebar does it also need to be in the list of linked items?

Reply:
I'd like to pull it out in another PR. We'll probably only need linked events in there.